app/utils/kcidb.py

```diff
@@ -245,11 +248,15 @@ def push_tests(group_id, kcidb_options, db_options={}, db=None):
                 },
                 'path': test['path'],
                 'description': test_description,
-                'status': test['status'],
+                'status': status_translate.get(test['status'], test['status']),
                 'waived': False,
                 'start_time': test['start_time'],
                 'output_files': output_files,
-                'misc': misc,
+                'misc': misc.copy().update(
```

Comment:
No I think the idea is to keep the original status value in `misc` rather than the translated one which is already in `status`.  So the `status_translate` dictionary shouldn't need to be used here.  Instead, the `kernelci-status` value can be put directly when initialising the `misc` directory with the value of `test['status']`.

Reply:
You're right I accidentally pushed the wrong version. My bad.
The question that still remains is if we want to have the `kernelci-status` added to `misc` only when the status was translated to something else or always. In case the latter is the correct solution, that indeed we can populate the value while creating the `misc` directory.